src/List.js

```diff
@@ -66,13 +86,33 @@ export function List() {
 
   const handleChange = async (id, event) => {
     let date = new Date();
+    const item = items.find((element) => element.id === id);
+    const daysSinceLastTransaction = item.lastPurchasedDate
+      ? Math.round((new Date() - item.lastPurchasedDate) / 1000 / 60 / 60 / 24)
```

Comment:
Maybe a variable here would help to make it more clear what `Math.round((new Date() - item.lastPurchasedDate) / 1000 / 60 / 60 / 24)` is.

Reply:
Thanks Mike, done!